docs/docs/recipes/styling-css.md

```diff
@@ -25,46 +22,45 @@ p {
   color: maroon;
 }
 ```
-
-2. Import the global CSS file in the `gatsby-browser.js` file such as the following:
+2. Importer la feuille de style CSS globale dans le fichier `gatsby-browser.js` comme suivant:
 
 ```javascript:title=gatsby-browser.js
 import "./src/styles/global.css"
 ```
 
-> **Note:** You can also make use of `require('./src/styles/global.css')` to import the global CSS file in your `gatsby-config.js` file.
+> **Note:** Vous pouvez aussi utiliser `require('./src/styles/global.css')` pour importer la feuille de style globale dans votre fichier gatsby-config.js`
 
-3. Run `gatsby-develop` to observe the global styling being applied across your site.
+3. Lancer `gatsby-develop` pour observer le style global appliqué à votre site.
```

Comment:
```suggestion
3. Lancez `gatsby-develop` pour observer le style global appliqué à votre site.
```

Reply:
Haha I was working on adding style on Gatsby at this moment so nice to read and learn while translating it!